src/Controller/EnMarche/MembershipController.php

```diff
@@ -33,6 +33,10 @@ class MembershipController extends Controller
      */
     public function registerAction(Request $request): Response
     {
+        if ($this->get('security.authorization_checker')->isGranted('IS_AUTHENTICATED_FULLY')) {
+            return $this->redirectToRoute('homepage');
```

Comment:
Why not redirect to the profile page instead?

Reply:
This question was asked. This is a specific request :)

Reply:
the guru changed his mind \o/